replicator/replicator.ps1

```diff
@@ -0,0 +1,26 @@
+$IsPresent = $false
+
+$TerraformDir = "$env:GITHUB_WORKSPACE/$env:TERRAFORM_DIR_PATH"
+
+if (Test-Path -Path "$TerraformDir/*.tf")
+{
+  Write-Information "Terraform found at $TerraformDir"
+  $IsPresent = $true
+  Invoke-EpsTemplate -Path $PSScriptRoot/templates/terraform_dynamodb.eps `
+  | Out-File -FilePath $TerraformDir/iaac_replicator_dynamodb.tf
+}
```

Comment:
Where are you running terraform apply to create entries in dynamo db able.

Reply:
Same place where helm is run

Reply:
Do you mean as part of deploy.ps1, thats where we run helm commands.

Reply:
Depends on the entrypoint in a project

Reply:
got it.

Reply:
https://octopus.apps.ops-drivevariant.com/app#/Spaces-2/projects/replicator/deployments/releases/0.1.0-add-octopus-v2-0001.319/deployments/Deployments-9239?tasklineid=0%2FServerTasks-50834_QNXHJYSJRL%2F1cec407d68664f27b4c1976f4d9d6c88%2F6bd8ddc7839a4fc18d486577350e2fd3&activeTab=taskLog